src/commands/convert.js

```diff
@@ -23,76 +27,331 @@ const options = {
+				multiplier: 10
+			},
+			{
+				symbol: 'hg',
+				names: ['hectogram', 'hectograms', 'hectomatre', 'hectogrammes'],
+				multiplier: 100
+			},
+			{
+				symbol: 'kg',
+				names: ['kilogram', 'kilograms', 'kilogramme', 'kilogrammes'],
+				multiplier: 1000
+			},
+			// Officially, it's 'megagram'. However, ton/tonne is also used and since both imperial and metric use it,
+			// 'Ton' will be imperial and 'Tonne' will be metric
+			{
+				symbol: 'Mg',
+				names: [
+					'metric ton', 'metric tons', 'megagram', 'megagrams', 'megagramme', 'megagrammes', 'tonne', 'tonnes', 'metric tonne', 'metric tonnes'
+				],
+				multiplier: 1000000
+			}
+		]
 	},
-	{
-		symbol: 'km',
-		names: ['kilometer', 'kilometers', 'kilometre', 'kilometres'],
-		multiplier: 1000
+	lengths: {
+		convert: sum => {
+			const inches = sum / inch;
+			const displayedUnits = imperialUnits.lengths.units.filter(unit =>
+				['ml', 'ft', 'in'].includes(unit.symbol)
+			);
+
+			// A special case to use the feet'inches" format when the number is
+			// between 1 and 15 feet
+			if (inches >= 12 && inches <= 180) {
+				const rounded = Math.round(inches);
+				return `${Math.floor(rounded / 12)}'${rounded % 12}"`;
+			}
+
+			return prefixUnit(displayedUnits, inches);
+		},
+		units: [
+			{
+				symbol: 'mm',
+				names: ['millimeter', 'millimeters', 'millimetre', 'millimetres'],
+				multiplier: 0.001
+			},
+			{
+				symbol: 'cm',
+				names: ['centimeter', 'centimeters', 'centimetre', 'centimetres'],
+				multiplier: 0.01
+			},
+			{
+				symbol: 'dm',
+				names: ['decimeter', 'decimeters', 'decimetre', 'decimetres'],
+				multiplier: 0.1
+			},
+			{
+				symbol: 'm',
+				names: ['meter', 'meters', 'metre', 'metres'],
+				multiplier: 1
+			},
+			{
+				symbol: 'dam',
+				names: ['decameter', 'decameters', 'decametre', 'decametres'],
+				multiplier: 10
+			},
+			{
+				symbol: 'hm',
+				names: ['hectometer', 'hectometers', 'hectomatre', 'hectometres'],
+				multiplier: 100
+			},
+			{
+				symbol: 'km',
+				names: ['kilometer', 'kilometers', 'kilometre', 'kilometres'],
+				multiplier: 1000
+			}
+		]
 	}
-];
+};
+
+const unitSystems = [...Object.values(imperialUnits), ...Object.values(metricUnits)];
+
+/**
+ * Deals with the conversion if it's simple enough.
+ * @param {number} sum The sum that is calculated in exec.
+ * @param {number} multiplier The number that should be multiplied to get the new unit.
+ * @param {Object} baseUnit A base unit object that contains all units. Example: metricUnits.weight.
+ * @param {Array<string>} specialUnits An array containing what kind of units should be shown as a result. This should contain non obscure units.
+ * @returns {number} A user-friendly representation of the value
+ */
+function simpleConvert(sum, multiplier, baseUnit, specialUnits) {
+	const result = sum * multiplier;
+	const displayedUnits = baseUnit.units.filter(unit => specialUnits.includes(unit.symbol));
```

Comment:
Is there any case where `displayedUnits` would be different from `specialUnits`? Couldn't we just give `specialUnits` to `prefixUnit`?

Reply:
No, wait so the goal is to get the unit object, right.

I'm starying to think that maybe it would be better if units were stored in a Map, where the key is the symbol, but that would require too much refactoring.